packages/manager/core/generator/app/templates/src/components/layout-helpers/Listing/dataGrid.tsx

```diff
@@ -36,7 +38,7 @@ const Datagrid: React.FC<Props> = ({ serviceKey, data }) => {
               </OsdsText>
             </th>
           ))}
-          <th>
+          <th className="p-5 bg-cyan-200">
```

Comment:
We need can maybe use ODS tokens for color
https://ovh.github.io/design-system/latest/?path=/story/code-design-tokens-blue-jeans-colors--page
Like `--ods-blue-jeans-color-primary-300`

Reply:
Agreed. Except the variable name is `--ods-color-primary-300`

Reply:
Yes, in the next weeks, we will delete this to replace by the ods component